src/dialogs/admin/RejectDialog.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <q-dialog v-model="rejectDialog" persistent>
+    <q-card
+      class="q-pa-md"
+      style="border-radius: 15px; background-color: black"
```

Comment:
change the background using quasar style classes
```suggestion
      class="q-pa-md bg-white rounded-borders"
```